frontend/tests/factories/QualityNominationFactory.php

```diff
@@ -50,12 +50,17 @@ public static function initTags() {
         );
     }
 
+    /**
+     * @param null|string[] $tags
+     * @return array{status: string, qualitynomination_id: int}
+     */
     public static function createSuggestion(
-        $login,
-        $problemAlias,
-        $difficulty,
-        $quality,
-        $tags
+        ScopedLoginToken $login,
+        string $problemAlias,
+        ?float $difficulty,
+        ?float $quality,
+        ?array $tags,
+        ?bool $beforeAC = false
```

Comment:
puedes quitarle el `= false`? así obligamos a todos los que llaman esta función a pasarle el parámetro.

también, no es necesario distinguir el valor nulo, así que puedes hacerlo no-nulo?

```suggestion
        bool $beforeAC
```